src/ui/components/Dashboard/RecordingItem/RecordingListItem.css

```diff
@@ -231,3 +231,7 @@
   background: var(--theme-body-color);
   margin: auto;
 }
+
+.page-url:hover {
+  color: var(--dark-blue) !important;
```

Comment:
Can we add `text-decoration: underline` here just to really drive the point home?

Reply:
sure